### PR TITLE
LPS-102645 Add test for treeview component

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by jbalsas** [comment]
@victorg1991, @p2kmgcl, do you have a sample usage for the new component? How can we test it visually?

File: modules/apps/frontend-taglib/frontend-taglib/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	extends: ['liferay/react'],
+	rules: {
+		'react/no-string-refs': 'off'
```
Comment (wincent) [inline modules/apps/frontend-taglib/frontend-taglib/.eslintrc.js]: Why do you need this?

**Comment by victorg1991** [comment]
I've updated a usage sample in SelectCategory.es.js

File: modules/apps/frontend-taglib/frontend-taglib/.eslintrc.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/frontend-taglib/frontend-taglib/.eslintrc.js]: If I remove that, when I run sourceFormat, there are some errors (this module mix soy and metal)

Comment (wincent) [reply modules/apps/frontend-taglib/frontend-taglib/.eslintrc.js]: > this module mix soy and metal

Got it. 👍

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/wincent/liferay-portal/pull/70. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**